python/nav/smidumps/__init__.py

```diff
@@ -3,3 +3,47 @@
 As dumped by smidump dump using the python format option.
 
 """
+from __future__ import absolute_import
+from os.path import dirname, join, basename, isfile, splitext
+import glob
+import importlib
+
+_submodule_files = (
+    glob.glob(join(dirname(__file__), "*.py")) +
+    glob.glob(join(dirname(__file__), "*.pyc")) +
+    glob.glob(join(dirname(__file__), "*.pyo"))
+)
+
+_submodules = list(set(
+    splitext(basename(f))[0]
+    for f in _submodule_files
+    if isfile(f) and not basename(f).startswith('__init__')
+))
+
+__all__ = _submodule_files = ['get_mib_modules', 'get_mib']
```

Comment:
why set _submodule_files here?

Reply:
Herp derp. An advanced typo. It's supposed to be 
`__all__ = _submodules + ['get_mib_modules', 'get_mib']`